source/store.go

```diff
@@ -81,12 +81,12 @@ type SingletonClientGenerator struct {
 	kubeClient      kubernetes.Interface
 	istioClient     *istioclient.Clientset
 	cfClient        *cfclient.Client
-	contourClient   dynamic.Interface
+	dynKubeClient   dynamic.Interface
```

Comment:
Why does this change from controur to dynKube?

Reply:
Because it is now used not only by contour but also by the Ambassador code, so I think that keeping it as `contourClient` could cause some confusion...